.github/workflows/static-analysis.yml

```
@@ -0,0 +1,129 @@
+          coverage: none
+
+      - name: Get composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Install dependencies
+        run: |
+          composer install --no-interaction --no-progress --prefer-dist
+
+      - name: Run composer-require-checker with config
+        if: ${{ inputs.has_crc_config }}
```

actionlint (1.7.4)

property "has_crc_config" is not defined in object type {php_version: any}

(expression)